lib/babel-preset.js

```diff
@@ -0,0 +1,21 @@
+exports.buildPreset = buildPreset;
+exports.default = buildPreset();
+Object.defineProperty(exports, '__esModule', { value: true });
+
+function buildPreset(options) {
+  const minNode = (options && options.minNode) || '4.3';
+
+  return {
+    presets: [['env', {
+      useBuiltIns: true,
+      targets: {
+        node: minNode,
+      },
+    }]],
+    plugins: [
+      ['transform-object-rest-spread', { useBuiltIns: true }],
+      ['transform-class-properties', { spec: true }],
```

Comment:
This is at `stage-2` @tc39 and I also doubt the babel implementation here, since _real_ privacy won't exists until the VM has actually implemented that in it's Interpreter